lib/Controller/SettingsController.php

```diff
@@ -39,11 +43,14 @@ public function __construct($AppName,
 	                            IURLGenerator $urlGenerator,
 	                            IL10N $trans,
 	                            ILogger $logger,
-	                            AppConfig $config
+	                            AppConfig $config,
+								IConfig $IConfig
```

Comment:
fix indentation

Reply:
the variable name is not adjusted with the PHP conventions. $IConfog should be changed to $appConfig or something like that.